### ISSUE TITLE
Cache

### ISSUE THREAD

**Issue by johnkramer2** [yuliskov/SmartTubeNext#584]
SmartTube Next Apk Keeps buffering on Shield TV Pro and I have to keep on clearing the cache and have to restart the app.

**Comment by Undead-Aquila** [comment]
I'm having issue since updating to version 11.

**Comment by johnkramer2** [comment]
Thanks for the heads up. I did change the buffer to high and that seemed to help a bit. The main issue seems bad when watching live streams. It 1st started with 10.75 but after the update to Version 11 it wasn't as bad.

**Comment by Undead-Aquila** [comment]
I was still have the issue on high, but once restarting the app it work perfect, then buffer problem happens after 30 minutes or so, there is a new update I got it through the app which seems to of fixed the buffering issue altogether.  It's version 11.3. Just to settings and about.

**Comment by johnkramer2** [comment]
Thank you so much. Just updated now. Can't wait to give it a try.

**Comment by Undead-Aquila** [comment]
Hope it works out. Been watching YouTube for 3 hours and not one buffer.

**Comment by hamswolo** [comment]
Yeah I'm having this issue on firestick 4k keeps buffering randomly.

**Comment by ACR-Jeff** [comment]
Marking this issue as Stale
